docs/tre-templates/workspace-services/gitea.md

```diff
@@ -0,0 +1,33 @@
+# Gitea Workspace Service
+
+See: [https://gitea.io/](https://gitea.io)
+
+## Firewall Rules
+
+The Gitea worskpace service needs outbound access to:
+
+- AzureActiveDirectory
+- Azure AD CDN - https://aadcdn.msftauth.net
+
+## Prerequisites
+
+- [A base workspace deployed](../workspaces/base.md)
+
+## Gitea Workspace Service Configuration
+
+When deploying a Gitea Workspace service the following properties need to be configured.
+
+| Property |  Description |
+| -------- |  ----------- |
+| `openid_client_id` | Valid client ID of the Workspace App Registration. |
```

Comment:
Should we use the variables that get spat out of the aad-reg script (WORKSPACE_API_CLIENT_ID, 
WORKSPACE_API_CLIENT_SECRET)

Reply:
I've used this as what is used in other bundles. Ultimately these values should be provided via the resource processor from the worksapce properties rather than needing manual input. 

I'll create a seperate issue to enable that, and rename the variables for all bundles.

Reply:
Added this issue, let me know if this makes sense. https://github.com/microsoft/AzureTRE/issues/1474